src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/unusedlocalvariable/InputUnusedLocalVariableNestedClasses4.java

```diff
@@ -0,0 +1,29 @@
+/*xml
+<module name="Checker">
+    <property name="charset" value="UTF-8" />
+    <property name="severity" value="warning" />
+    <property name="haltOnException" value="false" />
```

Comment:
why all of this matter ?

can it be simple:
```
/*
UnusedLocalVariable
*/
```

if can be removed, apply to all other input files.

Reply:
done